doc/CMakeLists.txt

```diff
@@ -1,12 +1,12 @@
 project(EnergyPlusDocs LANGUAGES NONE)
 
-cmake_minimum_required(VERSION 3.5.1)
+cmake_minimum_required(VERSION 3.12)
```

Comment:
In docs, we only need to bump to 3.12 for FindPython because PCH isn't in the mix... So I did 3.12 instead of 3.16, up for debate.

Reply:
Hmmm...I'm not sure.  I guess it's best to keep the minimum required to the minimum required for that specific project scope.  So 3.12 makes sense for the new FindPython.